install_files/ansible-base/roles/restore/tasks/main.yml

```diff
@@ -8,15 +8,15 @@
 
 - name: Copy backup tarball of config info to Application Server.
   copy:
-    src: "{{ restore_file }}"
-    dest: /tmp/{{ restore_file }}
+    src: /home/heartsucker/Downloads/sd-backup-2019-02-14--11-34-21.tar.gz #"{{ restore_file }}"
```

Comment:
Are these the values that were used for testing?

Reply:
lol crap yes :neutral_face: